pkg/services/alerting/notifiers/victorops.go

```diff
@@ -83,27 +83,6 @@ func (this *VictoropsNotifier) Notify(evalContext *alerting.EvalContext) error {
 		return nil
 	}
 
-	fields := make([]map[string]interface{}, 0)
```

Comment:
not sure I understand why all those rows where removed?

Reply:
fields is not used at all, or I am wrong with this?

Reply:
You're absolutely right.